app.py

```diff
@@ -143,8 +143,9 @@ def proposals():
     else:
         nodes = Node.query.all()
 
+    active_nodes = [node for node in nodes if node.is_active()]
```

Comment:
Nodes should be filtered in query layer
+
Dashboard *active filter* logic now is duplicated. Could be moved to reusable helpers

Reply:
Re-used database-level filtering.